### PR TITLE
Cli analyze - adding analyze to mta-cli analysis commands

### PR DESCRIPTION
**Pull request by emarcusRH**
Following comment from Alex in https://github.com/windup/windup-documentation/pull/828

updates to mta-cli commands in _docs/cli-guide/topics/cli-run.adoc_

### CODE REVIEW

File: docs/topics/cli-run.adoc [hunk 1]
```diff
@@ -11,11 +11,11 @@ You can run {ProductShortName} against your application.
 .Procedure
 
 . Open a terminal and navigate to the `<{ProductShortName}_HOME>/bin/` directory.
-. Execute the `windup-cli` script, or `windup-cli.bat` for Windows, and specify the appropriate arguments:
+. Execute the `{mta-cli}` script, or `{mta-cli}.bat` for Windows, and specify the appropriate arguments:
```
Comment (abrugaro) [inline docs/topics/cli-run.adoc:14]: In Windows, the mta-cli is an `.exe` file not a `.bat`

File: docs/topics/cli-run.adoc [hunk 2]
```diff
@@ -52,7 +52,7 @@ The following command analyzes the `org.jboss.seam` packages of the link:https:/
 
 [source,terminal,subs="attributes+"]
 ----
-$ <{ProductShortName}_HOME>/bin/windup-cli --sourceMode --input /path/to/seam-booking-5.2/ \
+$ <{ProductShortName}_HOME>/bin/{mta-cli} analyze --sourceMode --input /path/to/seam-booking-5.2/ \
```
Comment (abrugaro) [inline docs/topics/cli-run.adoc:55]: The `sourceMode` argumen has been replaced by `mode`

    -m, --mode string               analysis mode. Must be one of 'full' or 'source-only' (default "full")

File: docs/topics/cli-run.adoc [hunk 3]
```diff
@@ -76,7 +76,7 @@ To override the default _Fernflower_ decompiler, pass the `-Dwindup.decompiler`
 
 [source,terminal,subs="attributes+"]
 ----
-$ <{ProductShortName}_HOME>/bin/windup-cli -Dwindup.decompiler=procyon \
+$ <{ProductShortName}_HOME>/bin/{mta-cli} -Dwindup.decompiler=procyon \
```
Comment (abrugaro) [inline docs/topics/cli-run.adoc:79]: The `-D` argument no longer exists

File: docs/topics/cli-run.adoc [hunk 4]
```diff
@@ -11,11 +11,11 @@ You can run {ProductShortName} against your application.
 .Procedure
 
 . Open a terminal and navigate to the `<{ProductShortName}_HOME>/bin/` directory.
-. Execute the `windup-cli` script, or `windup-cli.bat` for Windows, and specify the appropriate arguments:
+. Execute the `{mta-cli}` script, or `{mta-cli}.bat` for Windows, and specify the appropriate arguments:
 +
 [source,terminal,subs="attributes+"]
 ----
-$ ./windup-cli --input /path/to/jee-example-app-1.0.0.ear \
+$ ./{mta-cli} --input /path/to/jee-example-app-1.0.0.ear \
```
Comment (abrugaro) [inline docs/topics/cli-run.adoc:18]: `analyze` is needed here

**Review by abrugaro — CHANGES_REQUESTED** [review]

File: docs/topics/cli-run.adoc (hunk 1 above)
Comment (emarcusRH) [reply docs/topics/cli-run.adoc:14]: yes, that is fixed in https://github.com/windup/windup-documentation/pull/828

File: docs/topics/cli-run.adoc (hunk 3 above)
Comment (emarcusRH) [reply docs/topics/cli-run.adoc:79]: I removed the entire section on overriding the decompiler , fixed in  https://github.com/windup/windup-documentation/pull/828

File: docs/topics/cli-run.adoc (hunk 2 above)
Comment (abrugaro) [reply docs/topics/cli-run.adoc]: My bad I didn't clarify, the example command would be 

    {mta-cli} analyze --mode source-only ...

**Comment by anarnold97** [comment]
@emarcusRH - fixing now

**Comment by emarcusRH** [comment]
these changes have been handled in a different PR - https://github.com/windup/windup-documentation/pull/847

**Pull request closed without merging**